apps/vanilla/src/global/variables.css

```
@@ -1,5 +1,30 @@
 :root {
-  --heading-color-1: #ffd700;
-  --heading-color-2: #e9967a;
-  --heading-color-3: #b0e0e6;
+  /* Colors */
+  --on-surface: rgb(0 0 0 / 100%);
+  --on-surface-dim: rgb(0 0 0 / 50%);
+  --surface: rgb(255 255 255 / 100%);
+  --surface-overlay: rgb(0 0 0 / 5%);
+  --primary: rgb(241 89 36 / 100%);
+  --on-primary: rgb(255 255 255);
+  --border: rgb(0 0 0 / 15%);
+
+  /* Font-size */
+  --font-size-md: 16px;
+  --font-size-lg: 18px;
+  --font-size-xl: 20px;
+
+  /* Spaces */
+  --space-2: 8px;
```

```suggestion
  --space-2: calc(--space-base * 2);
```

If we change `--space-base`, we won't need to change other variables.